src/hooks/useTrackVolume.ts

```diff
@@ -29,7 +29,8 @@ export function useTrackVolume(
 
   const mediaStreamTrack = track?.mediaStreamTrack;
   const hasMediaStreamTrack = mediaStreamTrack != null;
-  const peerConnectionId = mediaStreamTrack.peerConnectionId ?? -1;
+  // @ts-ignore - Accessing private property
+  const peerConnectionId = mediaStreamTrack._peerConnectionId ?? -1;
   const mediaStreamTrackId = mediaStreamTrack.id;
```

Comment:
```suggestion
  const mediaStreamTrack = track?.mediaStreamTrack as MediaStreamTrack | undefined;
  const hasMediaStreamTrack = mediaStreamTrack != null;
  const peerConnectionId = mediaStreamTrack?._peerConnectionId ?? -1;
  const mediaStreamTrackId = mediaStreamTrack?.id;
```

Ahh I see the problem here. `track?.mediaStreamTrack` is given back as an `any` type, so it wasn't checking the typing properly. Casting to the rn-webrtc type should reenable the type checking and allow us to remove the `@ts-ignore`.